account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -55,15 +56,65 @@ import type { PermitMessage, PermitDomain } from "../gas-manager.js";
  * });
  * ```
  *
- * @param {string | string[]} policyId the policyId (or list of policyIds) for Alchemy's gas manager
- * @returns {Pick<ClientMiddlewareConfig, "dummyPaymasterAndData" | "paymasterAndData">} partial client middleware configuration containing `dummyPaymasterAndData` and `paymasterAndData`
+ * @param {string | string[]} policyId - The policyId (or list of policyIds) for Alchemy's gas manager
+ * @param {PolicyToken | undefined} policyToken - The policy token configuration
+ * @returns {Pick<ClientMiddlewareConfig, "dummyPaymasterAndData" | "paymasterAndData">} Partial client middleware configuration containing `dummyPaymasterAndData` and `paymasterAndData`
  */
 export function alchemyGasManagerMiddleware(
-  policyId: string | string[]
+  policyId: string | string[],
+  policyToken: PolicyToken | undefined
```

Comment:
❗ i believe this needs to be optional to avoid breaking changes:
```suggestion
  policyToken?: PolicyToken
```